lightspark/src/objects/account_to_api_tokens_connection.rs

```diff
@@ -35,11 +41,15 @@ impl Connection for AccountToApiTokensConnection {
         self.page_info.clone()
     }
 
+
     fn type_name(&self) -> &'static str {
         "AccountToApiTokensConnection"
     }
 }
 
+
```

Comment:
Formatter - install cargo - https://doc.rust-lang.org/cargo/getting-started/installation.html

Reply:
Thanks! Fixed formatting, but seems cargo build is also failing. Looking as to why